### DISCUSSION TITLE
Find the position of the center of mass of the agent & obstacles.

### DISCUSSION THREAD

**Discussion by odyssey33** [PKU-Alignment/safety-gymnasium#54] [Q&A]
Hi,
How can I find the position of the agent and locations of the obstacles at every step? 
I'd like to figure out the distance between the agent and multiple obstacles.

**Comment by muchvo** [comment]
You can use env.task.[object_name].pos.
for example:
```python
env.task.agent.pos
env.task.goal.pos
env.task.vases.pos
```
Note:
	for multiple objects in same class, you should use plural nouns.

**Reply by muchvo** [reply]
And for now, we only support getting the position of the geometry center of objects via the method mentioned above.
